CHANGELOG.md

```diff
@@ -11,10 +11,11 @@ _Please add entries here for your pull requests that are not yet released._
 
 ### Breaking Changes
```

Comment:
Just require shakapacker v7 for v3 of react-rails so no older versions of webpacker, etc.

Reply:
Exactly